apps/extension/src/core/util/getConfig.ts

```diff
@@ -8,8 +9,15 @@ type Config = {
 
 export const CONFIG_RATE_LIMIT_ERROR = "Rate Limit Exceeded"
 
-export const getConfig = async (): Promise<Config> => {
-  const response = await fetch(TALISMAN_CONFIG_URL)
+export const getConfig = async (): Promise<Config | null> => {
+  try {
+    // eslint-disable-next-line no-var
+    var response = await fetch(TALISMAN_CONFIG_URL)
```

Comment:
Something to check - I don't think that `fetch` raises an error when the response is not `HTTP 200`.

So we might want another `if` which throws an error if [response.ok](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) is false.

Otherwise we'll see toml parser errors when the network / github is down.

Reply:
Correct, i'll add something for that